package.json

```diff
@@ -29,7 +29,7 @@
     "firebase": "npm run firebase-emulators:start",
     "firebase:all": "npm run firebase-functions:watch & npm run firebase-emulators:start",
     "firebase-export": "firebase emulators:export --force firebase_seed",
-    "firebase-emulators:start": "firebase emulators:start --project development --import=firebase_seed",
+    "firebase-emulators:start": "firebase emulators:start --import=firebase_seed",
```

Comment:
Firebase emulators will be connected to the project with "default" alias, that is project named "prime-deals-local"